.codecov.yml

```diff
@@ -24,3 +24,10 @@ comment:
   layout: "header, diff"
   behavior: default
   require_changes: no
+
+ignore:
+  - "**/*_generated.go"
+  - "test/**/*.go"
```

Comment:
I think codecov already ignores /vendor and /test, so these lines don't add anything?